packages/app/src/app/components/CreateNewSandbox/CreateSandbox/TemplateCard.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Stack, Element } from '@codesandbox/components';
+import { getTemplateIcon } from '@codesandbox/common/lib/utils/getTemplateIcon';
+
+import { TemplateFragment } from 'app/graphql/types';
+import { VisuallyHidden } from 'reakit/VisuallyHidden';
+import { TemplateButton } from './elements';
+
+export const TemplateCard = ({
+  template,
+  selectTemplate,
+}: {
+  template: TemplateFragment;
+  selectTemplate: (template: TemplateFragment) => void;
```

Comment:
nitpick: would name this `onSelectTemplate`. I was looking for the function to be implemented here otherwise